tests/p2p/bootstrap_test/test_restart_bootstrap.py

```diff
@@ -1,5 +1,52 @@
-# from tests.conftest import init_session
-from tests.test_bs import setup_bootstrap_in_namespace
+from pytest_testconfig import config as testconfig
+import time
+
+from tests.conftest import DeploymentInfo
+from tests import pod
+from tests.test_bs import setup_bootstrap_in_namespace, add_curl, setup_bootstrap
+from tests.misc import CoreV1ApiClient
+
+
+# reboot bootstrap
+# add a new stateful bootstrap
+# kill bootstrap
+# create a new bootstrap with the former key
+# validate state is the same as the old bootstrap node
+def test_reboot_bootstrap(init_session):
+    print("test_reboot_bootstrap")
+    # test params
+    sleep_time = 5
+    session_id = init_session
+    bootstrap_key = 'bootstrap'
+    bootstrap_group_id = 'bootstrap_key'
+    ss_file_path = "/Users/amit/workspace/go-spacemesh/tests/k8s/bootstrap-w-conf-ss.yml"
```

Comment:
Please move this constant to the beginning of the file and rename it to Capital letters.

Reply:
fixed in c74dfbdcb5e8306e68dfab3ca5eae9c2248c799a

Reply:
this test probably won't work in any other computer since it aims to your specific directory path. try using a relative path

Reply:
fixed in 207fd37bd289dcc691ce4b4207a102bc0f009af7